cpp/src/io/parquet/experimental/page_index_filter.cu

```diff
@@ -702,8 +703,12 @@ std::vector<std::vector<bool>> aggregate_reader_metadata::compute_data_page_mask
 
   auto const has_page_index =
     compute_has_page_index(per_file_metadata, row_group_indices, output_column_schemas);
-  CUDF_EXPECTS(has_page_index,
-               "Data page mask computation requires the Parquet page index for all output columns");
+
+  // TODO: Don't use page pruning in case of lists and structs until we support them
```

Comment:
This is currently failing for `list` types. Enabling a temporary read path without filtering data pages while we fix it.